pyproject.toml

```diff
@@ -75,6 +75,7 @@ dependencies = [
   "urllib3>=2.2.3",
   "xmltodict>=0.14.2",
   "openshift-python-wrapper~=11.0.10",
+  "marshmallow~=3.26.1",
```

Comment:
why we need it?

Reply:
it's one of the dependencies of `pytest-jira` and if updated to latest (4.0.0) it breaks pytest
```
TypeError: Field.__init__() got an unexpected keyword argument 'default'
```